_sections/30-projects.md

```diff
@@ -66,6 +66,7 @@ These projects pledge to drop Python 2 support in or before 2020.
 - [![](assets/metpy.png)MetPy](https://unidata.github.io/MetPy)
 - [![](assets/nengo.png)Nengo](https://www.nengo.ai/)
 - [![](assets/pylast.png)pyLast](https://github.com/pylast/pylast) <!-- url:https://github.com/pylast/pylast sg:314 -->
+- [![](assets/iris.png)iris](https://scitools.org.uk/iris/docs/latest/) <!-- url:https://github.com/SciTools/iris -->
```

Comment:
Not quite sure about the `sg:XXX`, what should be added here?

Reply:
@Carreau Thanks for the explanation for `sg`.... I've now already added the stargazers count for `iris` :smile: